components/admin_console/admin_definition.jsx

```diff
@@ -2438,7 +2451,6 @@ const AdminDefinition = {
                         label_default: 'Enable Open Server: ',
                         help_text: t('admin.team.openServerDescription'),
                         help_text_default: 'When true, anyone can signup for a user account on this server without the need to be invited.',
-                        isHidden: it.configIsTrue('ExperimentalSettings', 'RestrictSystemAdmin'),
```

Comment:
@jwilander Heads up this was restricted from before and I have removed it so they can now edit change if they want to have it as an Open Server.